db/seeds.rb

```diff
@@ -1,4 +1,5 @@
 jcasimir = User.create(username: "jcasimir", password: "test")
+admin = User.create(username: "admin", password: "admin", role: 1)
```

Comment:
good call. we wanted to have an admin in the prod db so we can play with that functionality. thoughts for getting around this? we can look into figaro tomorrow